docs/Developer-Guide_Build-Preparation.md

```diff
@@ -70,13 +70,13 @@ This is a typical board configuration:
 	MODULES_NEXT="brcmfmac bonding"
 	#
 	KERNEL_TARGET="legacy,current,dev"
-	CLI_TARGET="buster,bionic:current"
+	CLI_TARGET="buster,bionic,focal:current"
 	DESKTOP_TARGET="stretch:legacy,current"
 	
 	CLI_BETA_TARGET=""
 	DESKTOP_BETA_TARGET=""
 	#
-	RECOMMENDED="Ubuntu_bionic_legacy_desktop:90,Debian_stretch_current:100"
```

Comment:
This part is deprecated. 
More recent looks like this:
https://github.com/armbian/build/blob/master/config/boards/nanopiduo2.conf
This is quick-how to:
https://github.com/armbian/build/blob/master/config/boards/README.md